tools/simple_hydrog/lakes/cr_lake_files.F90

```diff
@@ -268,7 +270,8 @@ program cr_lake_files
    enddo
 enddo
 
-rcode= nf_close (ncid)
+CHECK_NF_ERRSTAT(nf_sync(ncid))
+CHECK_NF_ERRSTAT(nf_close(ncid))
```

Comment:
It appears `ncid` is associated with a file that is `NF_NOWRITE` and the sync may not be needed here.  Check the instances below in this same source file.